tmt/base.py

```diff
@@ -3564,6 +3564,20 @@ def plans(
             sources = None
 
         # Build the list, convert to objects, sort and filter
```

Comment:
@happz this comment should ideally move also to line #3580?

Reply:
I wouldn't think so, it describes the code below - collect plans, convert them to `Plan` instances, filter and so on.

Reply:
ok, as you wish.